frappe/desk/page/whats_new_page/whats_new_page.css

```diff
@@ -0,0 +1,284 @@
+.page-container{
```

Comment:
We use CSS variables extensively after the redesign. Values like font-size, colors, shadows should all come from CSS variables. Take a look at other .css files for reference.